CMakeLists.txt

```diff
@@ -476,6 +477,11 @@ check_c_source_compiles(
     MFPU_NEON_AVAILABLE FAIL_REGEX "not supported")
 set(CMAKE_REQUIRED_FLAGS)
 
+# FORCE_SSE2 option will only be shown if HAVE_SSE2_INTRIN is true
+if("${ARCH}" MATCHES "i[3-6]86")
+    cmake_dependent_option(FORCE_SSE2 "Always assume cpu is SSE2 capable" OFF "HAVE_SSE2_INTRIN" OFF)
```

Comment:
"CPU" should be capitalized as it's initialism.

Reply:
Fixed